src/gc/gc.d

```diff
@@ -1892,181 +1790,17 @@ struct Gcx
     {
         debug(PRINTF) printf("Minimizing.\n");
 
-        static bool isUsed(Pool *pool) nothrow
+        static bool isFree(Pool* p) pure nothrow { return p.freepages == p.npages; }
+        foreach (pool; pooltable.minimize!isFree())
```

Comment:
Using an alias template argument for isFree seems slightly over-enginieered. Just making isFree a method of Pool or defining it in PoolTable.minimize seems more natural to me.